aiida/cmdline/commands/cmd_import.py

```diff
@@ -17,7 +17,7 @@
 
 from aiida.cmdline.commands.cmd_verdi import verdi
 from aiida.cmdline.params import options
-from aiida.cmdline.params.types import GroupParamType, ImportPath
+from aiida.cmdline.params.types import GroupParamType, PathURL
```

Comment:
Two points about the name:

1. I did not immediately understand that this type basically allows a 'Path' _or_ 'URL'. But that is the only reason for its existence, since we have options that need to support both and Click does not support parameters of multiple type. So I think perhaps the name `PathOrUrl` and `FileOrUrl` would make this a lot easier to see.
2. Second point is already made implicitly in the previous. It is a matter of convention, but both [Microsoft](https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/141e06ef(v=vs.71)?redirectedfrom=MSDN) and [Google](https://google.github.io/styleguide/jsguide.html#naming-camel-case-defined) advocate to stick to camel-case (or pascal-case) rules, even when it comes to acronyms. So `URL` should become `Url`.

Reply:
Yeah... I guess what you would really want is to be able to pass multiple types, but that's not how the click API is built.
Besides concatenating Class names with `Or`, one can also just choose one of them (e.g. `File`). In that case, one then has to check the documentation to know exactly how our `File` implementation differs from the standard one.
Anyhow, I've used your suggestions for now. Just not entirely sure about the `FILEORURL` metavar... I guess it's good enough for the moment.